gateway/src/apicast/oauth/oidc.lua

```diff
@@ -30,20 +31,34 @@ local mt = {
 
 local empty = {}
 
-function _M.new(oidc_config)
+function _M.new(oidc_config, service)
```

Comment:
`service` is no longer needed, right?

Reply:
🤦‍♂ sorry, no needed but I didn't amend it.